docs/design/opm-tooling.md

```diff
@@ -124,15 +124,17 @@ In an effort to make channel head selection understandable and deterministic whe
 
 1. Add bundles to the underlying data store
 2. Choose the channel heads and default channel
-3. Rebuild the update graph starting at the new heads
-
-Channel head -- the "latest" operator in a channel -- selection is now informed by [semver](https://semver.org/). The heurstic is simple, the bundle with the highest version in each channel becomes the new head. The default channel is then taken from the maximum versioned bundle which defines a default channel.
+3. Rebuild the update graph starting at the new heads  
+<br />    
+Channel head -- the "latest" operator in a channel -- selection is now informed by [semver](https://semver.org/). The heurstic is that the bundle with the highest version in each channel becomes the new head. The default channel is then taken from the maximum versioned bundle which defines a default channel.
```

Comment:
It looks like this breaks the semver inline link when viewing this in the markdown preview tab.

Reply:
I _think_ adding inline html may also cause IDEs that have marker linters to complain too: https://github.com/updownpress/markdown-lint/blob/master/rules/033-no-inline-html.md.